middlewares/isLoggedIn.js

```diff
@@ -0,0 +1,36 @@
+const JWT = require('jsonwebtoken');
+
+exports.isLoggedin = async (req, res, next) => {
+	if (req.cookies?.accessToken) {
+		//access-token is available
+		try {
+			const verified = await JWT.verify(
+				req.cookies.accessToken,
+				process.env.JWT_ACCESS_SECRET
+			);
+			return next();
+		} catch (error) {
+			console.log('Access token invalid/expired');
+		}
+	}
+
+	console.log(
+		'Hello refresh-token checking as access token might be invalid/unavailable in cookies'
```

Comment:
Log message should be more formal and cover detail to debug application.
Suggestion: Please write more formal log message and omit words like "Hello"

Ref: https://www.dataset.com/blog/the-10-commandments-of-logging/